env/cloud/lambdas/terragrunt.hcl

```
@@ -102,22 +104,25 @@ dependency "s3" {
   mock_outputs_merge_strategy_with_state  = "shallow"
   mock_outputs_allowed_terraform_commands = ["init", "fmt", "validate", "plan", "show"]
   mock_outputs = {
-    reliability_file_storage_arn = null
-    vault_file_storage_arn       = null
-    vault_file_storage_id        = "placeholder"
-    archive_storage_arn          = null
-    archive_storage_id           = "placeholder"
-    lambda_code_arn              = null
-    lambda_code_id               = "placeholder"
+    reliability_file_storage_arn   = null
+    vault_file_storage_arn         = null
+    vault_file_storage_id          = "placeholder"
+    archive_storage_arn            = null
+    archive_storage_id             = "placeholder"
+    lambda_code_arn                = null
+    lambda_code_id                 = "placeholder"
+    audit_logs_archive_storage_id  = "placeholder"
+    audit_logs_archive_storage_arn = null
```

If you set this to a mock S3 bucket ARN, it should fix your `null` error:
```suggestion
    audit_logs_archive_storage_arn = "arn:aws:s3:::forms-mock-audit-logs-archive-storage"
```

---

It worked! Thanks :)